todo/users/models.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+import ormar
+from fastapi_users import models
+from fastapi_users.db import OrmarBaseUserModel, OrmarUserDatabase
+
+from ..db import BaseMeta
+
+
+class User(models.BaseUser):
+    first_name: Optional[str] = None
+    last_name: Optional[str] = None
+
+
+class UserCreate(models.BaseUserCreate):
+    first_name: Optional[str] = None
+    last_name: Optional[str] = None
+
+
+class UserUpdate(User, models.BaseUserUpdate):
+    pass
+
+
+class UserDB(User, models.BaseUserDB):
+    pass
```

Comment:
These four `User*` classes are horrible.  What are these models even abstracting?

Reply:
If these are because that's how SQLAlchemy does it, then I still hate SQLAlchemy.

Reply:
Hi! Those other models are used in the fastapi-user endpoints for registering, updating. I believe it's the fastapi-user lib that makes it redundant 🤷‍♂️